### ISSUE TITLE
`indentation` writes the indentation of a line an at-rule swallowed onto the end of that at-rule, and the file grows a level on every run

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#375]
### What minimal example or steps are needed to reproduce the bug?

Where the last node of a block is an at-rule carrying neither a block nor a semicolon of its own, that at-rule runs to the closing brace, so PostCSS files a comment written behind it into the at-rule's `raws.between` along with the whitespace. The comment is no node of the block, and its line is no line the tree has a raw of its own for.

`indentation` measures such a line all the same, reports it, and writes the fix into the at-rule's `raws.between`, at the front of it — which puts the whitespace on the at-rule's own line, several lines earlier in the file:

```css
a {
	@extend .b
	/* c */
}
```

Under `"tab"` that is reported at `3:2` as `Expected indentation of 2 tabs`, and `--fix` writes:

```css
a {
	@extend .b		
	/* c */
}
```

Two tabs at the end of the at-rule's line, the comment's own indentation untouched, and the warning standing again on the next run. So the file grows two tabs on every run of `--fix`, for ever, and never gets one character nearer what the option asks for.

Every syntax reads it the same way, plain CSS included, and both spellings of the option reach it. The run grows with the number of lines the at-rule swallowed: one comment adds two tabs on every run, two comments add three on the first run and four on each after it. Under a numeric option the whitespace written is spaces:

| option | input | run 1 | run 2 |
| --- | --- | --- | --- |
| `"tab"` | `a {\n\t@extend .b\n\t/* c */\n}` | `a {\n\t@extend .b\t\t\n\t/* c */\n}` | `a {\n\t@extend .b\t\t\t\t\n\t/* c */\n}` |
| `"tab"` | `a {\n\t@extend .b\n\t/* c */\n\t/* d */\n}` | `a {\n\t@extend .b\t\t\t\n\t…}` | `a {\n\t@extend .b\t\t\t\t\t\t\t\n\t…}` |
| `2` | `a {\n\t@extend .b\n\t/* c */\n}` | `a {\n  @extend .b    \n\t/* c */\n}` | `a {\n  @extend .b        \n\t/* c */\n}` |

The same block with the comment standing behind a declaration — `a {\n\tcolor: pink;\n\t/* c */\n}` — is a comment node of its own, is measured off its own `raws.before`, and settles on the first run.

### What minimal configuration is needed to reproduce the bug?

```json
{
	"plugins": ["@stylistic/stylelint-plugin"],
	"rules": {
		"@stylistic/indentation": "tab"
	}
}
```

### What did you expect to happen?

The indentation of the comment's line is written in front of that comment, wherever the parser filed the text around it, and one run of `--fix` settles the file. Where nothing the rule can write reaches that line, the warning stands and the file is left exactly as it was.

### What actually happened?

The whitespace is written several lines earlier, onto the end of the at-rule's own line, the fix going to the front of the raw the comment was swallowed into, and the file grows a level of indentation on every run without the reported line ever changing.

### Notes

Found while closing #292, which is the same swallowing seen from the closing brace: there the run in front of the brace stands in the at-rule's `raws.between` and three rules wrote the block's `raws.after`. This one is `indentation`'s alone and reproduces on `main` untouched by that branch.
